_posts/03-03-01-Namespaces.md

```diff
@@ -20,8 +20,11 @@ One recommended way to use namespaces is outlined in [PSR-4][psr4], which aims t
 namespace convention to allow plug-and-play code.
 
 In October 2014 the PHP-FIG deprecated the previous autoloading standard: [PSR-0][psr0], which has been replaced with
-[PSR-4][psr4]. Currently both are still usable, as PSR-4 requires PHP 5.3 and many PHP 5.2-only projects currently implement
-PSR-0. If you're going to use an autoloader standard for a new application or package then you almost certainly want
+[PSR-4][psr4]. Currently both are still perfectly usable and PSR-0 is not going away. As PSR-4 requires PHP 5.3 and
```

Comment:
Seriously, not going to have "But won't somebody think of the PHP 5.2" in here while PHP 7.0 is latest stable.